### ISSUE TITLE
Questions About SUES200 Test Performance

### ISSUE THREAD

**Issue by Frost24K** [C-water/SDPL_release#1]
Hi, 
    Thank you for providing this excellent work! After reading your paper and code, I want to ask few questions:

1. Are the performances of the various methods mentioned in Table 5 obtained by training on SUES200 and then testing on the test set?
2. If those methods were trained on SUES200 dataset, Could you please provide the training hyperparameters for each method?

**Comment by C-water** [comment]
Hi, 

MCCG[1] reported on the performance of various methods on SUES200, and we did not retrain other methods. There are two things to note: 1、Models are trained and tested at a single height; 2、We adopt SwinV2-B as the backbone network, and the parameters of the classifier are shown in Table 3.

If you use the data in Table 5, consider citing the paper MCCG[1]. Thank you very much!

[1] MCCG: A ConvNeXt-based multiple-classifier method for cross-view geo-localization.

**Comment by Frost24K** [comment]
Hi,

Thank you for your clear and detailed response!
